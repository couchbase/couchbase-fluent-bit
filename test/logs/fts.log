2021/03/09 17:32:15 cbauth: key: fts/remoteClients registered for tls config updates
2021-03-09T17:32:15.303+00:00 [INFO] main: /opt/couchbase/bin/cbft started (v0.6.0/5.5.0)
2021-03-09T17:32:15.304+00:00 [INFO] main: file descriptor limit current: 1048576 max: 1048576
2021-03-09T17:32:15.304+00:00 [INFO]   -authType="cbauth"
2021-03-09T17:32:15.304+00:00 [INFO]   -bindGrpc="cb-example-0000.cb-example.default.svc:9130,0.0.0.0:9130"
2021-03-09T17:32:15.305+00:00 [INFO] main: registered bleve stores
2021-03-09T17:32:15.305+00:00 [INFO]   metrics
2021-03-09T17:32:15.305+00:00 [INFO] main: curr dir: "/opt/couchbase/var/lib/couchbase"
2021-03-09T17:32:15.357+00:00 [WARN] janitor: JanitorOnce, err: janitor: skipped on nil planPIndexes -- cbgt.(*Manager).JanitorLoop() at manager_janitor.go:98
2021-03-09T17:32:15.365+00:00 [INFO] Using plain authentication for user <ud>@fts</ud> 
2021-03-09T17:37:15.604+00:00 [INFO] managerStats: {"feeds":{},"pindexes":{},"manager":{"TotKick":0,"TotSetOptions":0,"TotRegisterFeed":0,"TotUnregisterFeed":0,"TotRegisterPIndex":0,"TotUnregisterPIndex":0,"TotSaveNodeDef":2,"TotSaveNodeDefNil":0,"TotSaveNodeDefGetErr":0,"TotSaveNodeDefSetErr":0,"TotSaveNodeDefRetry":0,"TotSaveNodeDefSame":0,"TotSaveNodeDefOk":2,"TotCreateIndex":0,"TotCreateIndexOk":0,"TotDeleteIndex":0,"TotDeleteIndexOk":0,"TotIndexControl":0,"TotIndexControlOk":0,"TotDeleteIndexBySource":0,"TotDeleteIndexBySourceErr":0,"TotDeleteIndexBySourceOk":0,"TotPlannerOpStart":0,"TotPlannerOpRes":0,"TotPlannerOpErr":0,"TotPlannerOpDone":0,"TotPlannerNOOP":0,"TotPlannerNOOPOk":0,"TotPlannerKick":0,"TotPlannerKickStart":0,"TotPlannerKickChanged":0,"TotPlannerKickErr":0,"TotPlannerKickOk":0,"TotPlannerUnknownErr":0,"TotPlannerSubscriptionEvent":0,"TotPlannerStop":0,"TotJanitorOpStart":4,"TotJanitorOpRes":4,"TotJanitorOpErr":4,"TotJanitorOpDone":4,"TotJanitorNOOP":0,"TotJanitorNOOPOk":0,"TotJanitorKick":4,"TotJanitorKickStart":4,"TotJanitorKickErr":4,"TotJanitorKickOk":0,"TotJanitorClosePIndex":0,"TotJanitorRemovePIndex":0,"TotJanitorRestartPIndex":0,"TotJanitorLoadDataDir":0,"TotJanitorUnknownErr":0,"TotJanitorSubscriptionEvent":3,"TotJanitorStop":0,"TotRefreshLastNodeDefs":6,"TotRefreshLastIndexDefs":2,"TotRefreshLastPlanPIndexes":0}}
2021-03-09T17:37:15.604+00:00 [INFO] stats: {
    "batch_bytes_added": 0,
    "batch_bytes_removed": 0,
    "curr_batches_blocked_by_herder": 0,
    "num_bytes_used_ram": 72349944,
    "pct_cpu_gc": 0.000023606166605874832,
    "tot_batches_flushed_on_maxops": 0,
    "tot_batches_flushed_on_timer": 0,
    "tot_bleve_dest_closed": 0,
    "tot_bleve_dest_opened": 0,
    "tot_grpc_listeners_closed": 0,
    "tot_grpc_listeners_opened": 1,
    "tot_grpc_queryreject_on_memquota": 0,
    "tot_grpcs_listeners_closed": 1,
    "tot_grpcs_listeners_opened": 2,
    "tot_http_limitlisteners_closed": 0,
    "tot_http_limitlisteners_opened": 1,
    "tot_https_limitlisteners_closed": 1,
    "tot_https_limitlisteners_opened": 2,
    "tot_queryreject_on_memquota": 0,
    "tot_remote_grpc": 0,
    "tot_remote_grpc_tls": 0,
    "tot_remote_http": 0,
    "tot_remote_http2": 0,
    "total_gc": 5,
    "total_queries_rejected_by_herder": 0
}
2021-03-09T17:42:15.567+00:00 [INFO] managerStats: {"feeds":{},"pindexes":{},"manager":{"TotKick":0,"TotSetOptions":0,"TotRegisterFeed":0,"TotUnregisterFeed":0,"TotRegisterPIndex":0,"TotUnregisterPIndex":0,"TotSaveNodeDef":2,"TotSaveNodeDefNil":0,"TotSaveNodeDefGetErr":0,"TotSaveNodeDefSetErr":0,"TotSaveNodeDefRetry":0,"TotSaveNodeDefSame":0,"TotSaveNodeDefOk":2,"TotCreateIndex":0,"TotCreateIndexOk":0,"TotDeleteIndex":0,"TotDeleteIndexOk":0,"TotIndexControl":0,"TotIndexControlOk":0,"TotDeleteIndexBySource":0,"TotDeleteIndexBySourceErr":0,"TotDeleteIndexBySourceOk":0,"TotPlannerOpStart":0,"TotPlannerOpRes":0,"TotPlannerOpErr":0,"TotPlannerOpDone":0,"TotPlannerNOOP":0,"TotPlannerNOOPOk":0,"TotPlannerKick":0,"TotPlannerKickStart":0,"TotPlannerKickChanged":0,"TotPlannerKickErr":0,"TotPlannerKickOk":0,"TotPlannerUnknownErr":0,"TotPlannerSubscriptionEvent":0,"TotPlannerStop":0,"TotJanitorOpStart":4,"TotJanitorOpRes":4,"TotJanitorOpErr":4,"TotJanitorOpDone":4,"TotJanitorNOOP":0,"TotJanitorNOOPOk":0,"TotJanitorKick":4,"TotJanitorKickStart":4,"TotJanitorKickErr":4,"TotJanitorKickOk":0,"TotJanitorClosePIndex":0,"TotJanitorRemovePIndex":0,"TotJanitorRestartPIndex":0,"TotJanitorLoadDataDir":0,"TotJanitorUnknownErr":0,"TotJanitorSubscriptionEvent":3,"TotJanitorStop":0,"TotRefreshLastNodeDefs":6,"TotRefreshLastIndexDefs":2,"TotRefreshLastPlanPIndexes":0}}
