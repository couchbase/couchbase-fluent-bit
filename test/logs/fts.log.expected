/fluent-bit/test/logs/fts.log.actual: [__IGNORED__, {"filename":"/fluent-bit/test/logs/fts.log","timestamp":"2021/03/09 17:32:15","level":"cbauth","message":" key: fts/remoteClients registered for tls config updates","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"}}]
/fluent-bit/test/logs/fts.log.actual: [__IGNORED__, {"filename":"/fluent-bit/test/logs/fts.log","timestamp":"2021-03-09T17:32:15.303+00:00","message":" main: /opt/couchbase/bin/cbft started (v0.6.0/5.5.0)","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"INFO"}]
/fluent-bit/test/logs/fts.log.actual: [__IGNORED__, {"filename":"/fluent-bit/test/logs/fts.log","timestamp":"2021-03-09T17:32:15.304+00:00","message":" main: file descriptor limit current: 1048576 max: 1048576","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"INFO"}]
/fluent-bit/test/logs/fts.log.actual: [__IGNORED__, {"filename":"/fluent-bit/test/logs/fts.log","timestamp":"2021-03-09T17:32:15.304+00:00","message":"   -authType=\"cbauth\"","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"INFO"}]
/fluent-bit/test/logs/fts.log.actual: [__IGNORED__, {"filename":"/fluent-bit/test/logs/fts.log","timestamp":"2021-03-09T17:32:15.304+00:00","message":"   -bindGrpc=\"cb-example-0000.cb-example.default.svc:9130,0.0.0.0:9130\"","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"INFO"}]
/fluent-bit/test/logs/fts.log.actual: [__IGNORED__, {"filename":"/fluent-bit/test/logs/fts.log","timestamp":"2021-03-09T17:32:15.305+00:00","message":" main: registered bleve stores","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"INFO"}]
/fluent-bit/test/logs/fts.log.actual: [__IGNORED__, {"filename":"/fluent-bit/test/logs/fts.log","timestamp":"2021-03-09T17:32:15.305+00:00","message":"   metrics","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"INFO"}]
/fluent-bit/test/logs/fts.log.actual: [__IGNORED__, {"filename":"/fluent-bit/test/logs/fts.log","timestamp":"2021-03-09T17:32:15.305+00:00","message":" main: curr dir: \"/opt/couchbase/var/lib/couchbase\"","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"INFO"}]
/fluent-bit/test/logs/fts.log.actual: [__IGNORED__, err: janitor: skipped on nil planPIndexes -- cbgt.(*Manager).JanitorLoop() at manager_janitor.go:98","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"WARN"}]
/fluent-bit/test/logs/fts.log.actual: [__IGNORED__, {"filename":"/fluent-bit/test/logs/fts.log","timestamp":"2021-03-09T17:32:15.365+00:00","message":" Using plain authentication for user <ud>@fts</ud> ","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"INFO"}]
/fluent-bit/test/logs/fts.log.actual: [__IGNORED__, {"filename":"/fluent-bit/test/logs/fts.log","timestamp":"2021-03-09T17:37:15.604+00:00","message":" managerStats: {\"feeds\":{},\"pindexes\":{},\"manager\":{\"TotKick\":0,\"TotSetOptions\":0,\"TotRegisterFeed\":0,\"TotUnregisterFeed\":0,\"TotRegisterPIndex\":0,\"TotUnregisterPIndex\":0,\"TotSaveNodeDef\":2,\"TotSaveNodeDefNil\":0,\"TotSaveNodeDefGetErr\":0,\"TotSaveNodeDefSetErr\":0,\"TotSaveNodeDefRetry\":0,\"TotSaveNodeDefSame\":0,\"TotSaveNodeDefOk\":2,\"TotCreateIndex\":0,\"TotCreateIndexOk\":0,\"TotDeleteIndex\":0,\"TotDeleteIndexOk\":0,\"TotIndexControl\":0,\"TotIndexControlOk\":0,\"TotDeleteIndexBySource\":0,\"TotDeleteIndexBySourceErr\":0,\"TotDeleteIndexBySourceOk\":0,\"TotPlannerOpStart\":0,\"TotPlannerOpRes\":0,\"TotPlannerOpErr\":0,\"TotPlannerOpDone\":0,\"TotPlannerNOOP\":0,\"TotPlannerNOOPOk\":0,\"TotPlannerKick\":0,\"TotPlannerKickStart\":0,\"TotPlannerKickChanged\":0,\"TotPlannerKickErr\":0,\"TotPlannerKickOk\":0,\"TotPlannerUnknownErr\":0,\"TotPlannerSubscriptionEvent\":0,\"TotPlannerStop\":0,\"TotJanitorOpStart\":4,\"TotJanitorOpRes\":4,\"TotJanitorOpErr\":4,\"TotJanitorOpDone\":4,\"TotJanitorNOOP\":0,\"TotJanitorNOOPOk\":0,\"TotJanitorKick\":4,\"TotJanitorKickStart\":4,\"TotJanitorKickErr\":4,\"TotJanitorKickOk\":0,\"TotJanitorClosePIndex\":0,\"TotJanitorRemovePIndex\":0,\"TotJanitorRestartPIndex\":0,\"TotJanitorLoadDataDir\":0,\"TotJanitorUnknownErr\":0,\"TotJanitorSubscriptionEvent\":3,\"TotJanitorStop\":0,\"TotRefreshLastNodeDefs\":6,\"TotRefreshLastIndexDefs\":2,\"TotRefreshLastPlanPIndexes\":0}}","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"INFO"}]
/fluent-bit/test/logs/fts.log.actual: [__IGNORED__, {"filename":"/fluent-bit/test/logs/fts.log","timestamp":"2021-03-09T17:37:15.604+00:00","message":" stats: {\n    \"batch_bytes_added\": 0,\n    \"batch_bytes_removed\": 0,\n    \"curr_batches_blocked_by_herder\": 0,\n    \"num_bytes_used_ram\": 72349944,\n    \"pct_cpu_gc\": 0.000023606166605874832,\n    \"tot_batches_flushed_on_maxops\": 0,\n    \"tot_batches_flushed_on_timer\": 0,\n    \"tot_bleve_dest_closed\": 0,\n    \"tot_bleve_dest_opened\": 0,\n    \"tot_grpc_listeners_closed\": 0,\n    \"tot_grpc_listeners_opened\": 1,\n    \"tot_grpc_queryreject_on_memquota\": 0,\n    \"tot_grpcs_listeners_closed\": 1,\n    \"tot_grpcs_listeners_opened\": 2,\n    \"tot_http_limitlisteners_closed\": 0,\n    \"tot_http_limitlisteners_opened\": 1,\n    \"tot_https_limitlisteners_closed\": 1,\n    \"tot_https_limitlisteners_opened\": 2,\n    \"tot_queryreject_on_memquota\": 0,\n    \"tot_remote_grpc\": 0,\n    \"tot_remote_grpc_tls\": 0,\n    \"tot_remote_http\": 0,\n    \"tot_remote_http2\": 0,\n    \"total_gc\": 5,\n    \"total_queries_rejected_by_herder\": 0\n}","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"INFO"}]
/fluent-bit/test/logs/fts.log.actual: [__IGNORED__, {"filename":"/fluent-bit/test/logs/fts.log","timestamp":"2021-03-09T17:42:15.567+00:00","message":" managerStats: {\"feeds\":{},\"pindexes\":{},\"manager\":{\"TotKick\":0,\"TotSetOptions\":0,\"TotRegisterFeed\":0,\"TotUnregisterFeed\":0,\"TotRegisterPIndex\":0,\"TotUnregisterPIndex\":0,\"TotSaveNodeDef\":2,\"TotSaveNodeDefNil\":0,\"TotSaveNodeDefGetErr\":0,\"TotSaveNodeDefSetErr\":0,\"TotSaveNodeDefRetry\":0,\"TotSaveNodeDefSame\":0,\"TotSaveNodeDefOk\":2,\"TotCreateIndex\":0,\"TotCreateIndexOk\":0,\"TotDeleteIndex\":0,\"TotDeleteIndexOk\":0,\"TotIndexControl\":0,\"TotIndexControlOk\":0,\"TotDeleteIndexBySource\":0,\"TotDeleteIndexBySourceErr\":0,\"TotDeleteIndexBySourceOk\":0,\"TotPlannerOpStart\":0,\"TotPlannerOpRes\":0,\"TotPlannerOpErr\":0,\"TotPlannerOpDone\":0,\"TotPlannerNOOP\":0,\"TotPlannerNOOPOk\":0,\"TotPlannerKick\":0,\"TotPlannerKickStart\":0,\"TotPlannerKickChanged\":0,\"TotPlannerKickErr\":0,\"TotPlannerKickOk\":0,\"TotPlannerUnknownErr\":0,\"TotPlannerSubscriptionEvent\":0,\"TotPlannerStop\":0,\"TotJanitorOpStart\":4,\"TotJanitorOpRes\":4,\"TotJanitorOpErr\":4,\"TotJanitorOpDone\":4,\"TotJanitorNOOP\":0,\"TotJanitorNOOPOk\":0,\"TotJanitorKick\":4,\"TotJanitorKickStart\":4,\"TotJanitorKickErr\":4,\"TotJanitorKickOk\":0,\"TotJanitorClosePIndex\":0,\"TotJanitorRemovePIndex\":0,\"TotJanitorRestartPIndex\":0,\"TotJanitorLoadDataDir\":0,\"TotJanitorUnknownErr\":0,\"TotJanitorSubscriptionEvent\":3,\"TotJanitorStop\":0,\"TotRefreshLastNodeDefs\":6,\"TotRefreshLastIndexDefs\":2,\"TotRefreshLastPlanPIndexes\":0}}","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"INFO"}]
